README.md

```diff
@@ -19,6 +19,10 @@ There is also an example app under `packages/example-app` whose goal is to demon
 
 - clone the project: `git clone https://github.com/bamlab/react-native-project-config`,
 - install the dependencies: `yarn install`,
+- install the dependencies in the example-app: `cd example-app && yarn && cd ../`
+- build the eslint-plugin: `cd packages/eslint-plugin && yarn build && cd ../..`
```

Comment:
You can replace with one command: `yarn workspace @bam.tech/eslint-plugin build`